ouster-ros/launch/sensor.independent.launch.xml

```diff
@@ -44,6 +44,17 @@
   <arg name="lidar_frame" default="os_lidar" description="value can not be empty"/>
   <arg name="imu_frame" default="os_imu" description="value can not be empty"/>
 
+  <arg name="use_system_default_qos" default="false" description="Use the default system QoS settings"/>
+  <arg name="proc_mask" default="IMG|PCL|IMU|SCAN" description="
+    currently IMG is not supported and does not affect anything,
```

Comment:
specifically for this launch?

Reply:
Yes, specifically for this launch file. `sensor.independent.launch.xml` still use the `os_sensor`, `os_cloud`, and `os_image`. The `os_cloud` can only control `imu`, `cloud` and `scan` messages. It can not control image messages, thus to remove image messages you would have to comment out the `os_image` node. The other launch file `sensor.composite.launch.xml` replaces the composition of 3 components with `os_driver` which supports the 4 flags.

Reply:
Yes, specifically for this launch file. `sensor.independent.launch.xml` still use the `os_sensor`, `os_cloud`, and `os_image`. The `os_cloud` can only control `imu`, `cloud` and `scan` messages. It can not control image messages, thus to remove image messages you would have to comment out the `os_image` node. The other launch file `sensor.composite.launch.xml` replaces the composition of 3 components with `os_driver` which supports the 4 flags.